### DISCUSSION TITLE
Best workflow for logging SOTA QSOs as an activator (not a chaser)

### DISCUSSION THREAD

**Discussion by aentinger** [magicbug/Cloudlog#1495] [Q&A] [answered]
Hi ;)

I've played around with Cloudlog a little and it seems to me that the workflow for logging QSOs as a SOTA activator (so as the guy on top of the summit) would be to:

- Create a separate station profile for each summit
- Log all QSOs for that summit under said profile (only enter a SOTA reference if it is a Summit-2-Summit QSO)

In this case the exported ADIF file is correctly parsed by the SOTA database's [ADIF upload feature](https://www.sotadata.org.uk/de/upload/activator/adif).

Is there a better way to do this?

Cheers,
vy 73 de Alex, OE5LXR

**Comment by ryandzink** [comment, marked as answer]
👍  That's exactly how I do it, with the result being that I have a lot of station profiles. Ever since SOTA added the ADIF upload in addition to the CSV upload I have only ever used ADIF and this workflow works well. If you happen to be on a SOTA summit in a POTA location/WWFF location (not sure if POTA has made its way to your area) you can have the station location represent both by putting in the Signature and Signature information fields and the exported ADIF files will be valid for both programs.

Ryan KI7QEK

**Reply by aentinger** [reply]
✔️ Thank you for clarifying this. I thought this method a bit cumbersome and wondered if there might be a more streamlined way to do it. Investigating the software I've noticed that it builds around the idea of different "station locations" as a central concept, so I guess that's the way to go 👍
